src/Fetcher/TululaFetcher.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
+
+use App\Entity\Conference;
+use App\Entity\Continent;
+use App\Entity\ExcludedTag;
+use App\Entity\Tag;
+use Doctrine\Persistence\ManagerRegistry;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\String\Slugger\AsciiSlugger;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TululaFetcher implements FetcherInterface
+{
+    const SOURCE = 'tulula';
+    const TAGS_SYNONYMS = [
```

Comment:
Looks like it could be private?

Reply:
The naming seems broken. It's not about synonyms, but about filters, isn't?

Reply:
I'm not too sure about this one, maybe we should wait @welcoMattic since he wrote it ?